addon/libconiks.so

```diff
@@ -0,0 +1 @@
+../chrome/content/libconiks.so
```

Comment:
It's green finally. I have to change directory many times which looks not really good, though.